src/main/kotlin/com/openlattice/postgres/external/ExternalDatabaseConnectionManager.kt

```diff
@@ -0,0 +1,62 @@
+package com.openlattice.postgres.external
+
+import com.google.common.cache.CacheBuilder
+import com.google.common.cache.CacheLoader
+import com.google.common.cache.LoadingCache
+import com.openlattice.assembler.AssemblerConfiguration
+import com.openlattice.assembler.PostgresDatabases
+import com.zaxxer.hikari.HikariConfig
+import com.zaxxer.hikari.HikariDataSource
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+/**
+ * @author Drew Bailey &lt;drew@openlattice.com&gt;
+ */
+class ExternalDatabaseConnectionManager(
+        private val assemblerConfiguration: AssemblerConfiguration
+) {
+
+    private val perDbCache: LoadingCache<String, HikariDataSource> = CacheBuilder
+            .newBuilder()
+            .expireAfterAccess(1, TimeUnit.HOURS)
+            .build(cacheLoader())
+
+    fun createOrgDataSource( organizationId: UUID ): HikariDataSource {
```

Comment:
oh huh I assumed this was a helper for `connect` -- in that case I'd be in favor of adding like a 
```
fun connectToOrg( organizationId: UUID ) {
  return connect( PostgresDatabases.buildOrganizationDatabaseName(organizationId) )
}
```
and using that basically everywhere that we use`connect` after calling `buildOrganizationDatabaseName`

Reply:
(generally in the interest of removing the how-do-i-map-from-OL-ids-to-specific-atlas-formats logic wherever possible and isolating that to one place, but at the very least this seems like a super simple case that will achieve that + reduce amount of code + make it cleaner)

Reply:
yep, has been added and used now